src/DataGrid.php

```diff
@@ -1360,22 +1360,32 @@ public function setFilter(array $filter): self
 	public function setDefaultFilter(array $defaultFilter, bool $useOnReset = true): self
 	{
 		foreach ($defaultFilter as $key => $value) {
+			/** @var Filter|null $filter */
 			$filter = $this->getFilter($key);
 
+			if ($filter === null) {
+				throw new DataGridException("Can not set default value to nonexisting filter [$key]");
+			}
+
 			if ($filter instanceof FilterMultiSelect && !is_array($value)) {
 				throw new DataGridException(
-					sprintf('Default value of filter [%s] - MultiSelect has to be an array', $key)
+					"Default value of filter [$key] - MultiSelect has to be an array"
```

Comment:
Why you feel like coding style changes are needed?

Reply:
fixed